### DISCUSSION TITLE
code: 2, kind: NotFound, message: "No such file or directory"

### DISCUSSION THREAD

**Discussion by LegendarySaiyan** [PyO3/pyo3#3047] [Questions]
Hi! In my lib I have two fonts. In the pdf_builder.rs file I use the font path to set the fonts for printpdf library. Like this:

```
let arial = doc.add_external_font(File::open("./src/fonts/Arial.ttf").unwrap()).unwrap();
let arial_bold = doc.add_external_font(File::open("./src/fonts/Arial Bold.ttf").unwrap()).unwrap();
```

But when I install the lib from pip, and try to launch the code, I get this error:

code: 2, kind: NotFound, message: "No such file or directory"

Is there any way to leave the fonts inside the rust library? It is really unwanted to pass a path from the python module.

**Comment by mejrs** [comment]
Consider using `include_bytes!` to bundle these at compile time.

**Reply by LegendarySaiyan** [reply]
Many thanks :)

**Comment by birkenfeld** [comment]
And please use discussions for such questions.

**Reply by LegendarySaiyan** [reply]
Thank you gentleman. Excuse me for questions here :)
